SU-SWS/saa_victoria_subtheme
# ADAPT-481: Adjusting margin on top of tables after headings.

# READY FOR REVIEW

# Summary
- Set margin top to be 0 for tables following an h2 or h3.

# Review By (Date)
- 7/28

# Urgency
- low priority, but noticed on current site: https://saaassociates.sites.stanford.edu/board-governors 

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. create a table immediately following an h2 and one immediately following an h3 and check spacing between.

![image](https://user-images.githubusercontent.com/12898896/88595786-e5bb2700-d020-11ea-8fe3-76b7f8077410.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-481
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
